### ISSUE TITLE
Kimai installer not running

### ISSUE THREAD

**Issue by matthewd333** [kimai/kimai#843]
Hello,

Kimai looks fantastic. You are really doing an amazing job to create this. Using my beginner knowledge I have managed to get Kimai installed using XAMPP.

But I am failing to install it on my Synology DS Server.  When I try running the installer: localdrive/home/www/html/kimai/installer/index.php the logo appears and some random snippets of code:
. */ $installsteps = 8; $kga = array(); require("../includes/version.php"); ?> 
"; } $width = $i * 15; echo ""; ?>

...but no action.

(see screen capture)
![capture](https://cloud.githubusercontent.com/assets/22421055/18811820/f8a325cc-831a-11e6-9a96-0f5aa8c628fa.PNG)

I have the following services installed and running on the diskstation;
php 5.6
phpMyAdmin
MaraDB
Web Station

Pointers would be much appreciated.

**Comment by simonschaufi** [comment]
file:// is not a valid protocoll. You need http or https.

**Comment by matthewd333** [comment]
Forgive my inexperience but when use http:// or https:// I get 404 not found.

**Comment by simonschaufi** [comment]
how about `http://localhost/kimai/` ?

**Comment by matthewd333** [comment]
Thanks,

Thats what I used for XAMPP

Using the ip address works for the DS under the default web root:
http://192.168.2.2/kimai/installer/index.php

Actually the server name works too.

I think my problem was I was trying to use Web Station root instead of the default web root.

Cheers for the prompt reply. You are doing a great job! I can't wait to have a play on Kimai

I made my own excel macro based user-form application for time logging when I was learning VBA programming. Kimai looks a lot better and I am now learning something new using SQL
